modules/xmpp/RtxModifier.spec.js

```diff
@@ -298,6 +298,20 @@ describe('RtxModifier', () => {
             });
         });
 
+        describe('when given an sdp with a flexfec stream', () => {
+            beforeEach(function() {
+                this.multipleVideoSdp = SampleSdpStrings.flexFecSdp;
```

Comment:
Is it necessary to put flexFecSdp onto an instance variable? It looks like it might be okay to pass in directly to transform.write.